lib/gestures.dart

```diff
@@ -1,5 +1,41 @@
 import 'package:flutter/gestures.dart';
 
+// Multi touch detector
```

Comment:
is it better to keep them after the basics? like it's sorta done on the other files?

Reply:
I am not sure if the order matter here, I didn't put the advanced one first for any particular reason.